features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/services/login/login_callback.jag

```diff
@@ -135,12 +140,12 @@
     if (state !== null) {
         state = decodeURIComponent(state);
         if ( state !== '/') {
-            response.sendRedirect(site.context + state);
+            response.sendRedirect(app.context + state);
```

Comment:
Same here. A proxy server can also rewrite redirections: Eg: (nginx)

`proxy_redirect https://localhost:9443/publisher https://localhost/custom-publisher;`

Reply:
+1, only the callback URL needs to be populated using the origin, as it goes as a query param.